README.md

```diff
@@ -295,6 +296,18 @@ test('passes when value passes given predicate', () => {
 });
 ```
 
+#### .toSatisfyAll(predicate)
+
+Use `.toSatisfyAll` when you want to use a custom matcher by supplying a predicate function that returns a `Boolean` for all values in an array.
+
+```js
+test('passes when all values in array pass given predicate', () => {
+  const isOdd = el => el % 2 === 1;
+  expect([1,3,5,7]).toSatisfy(isOdd);
```

Comment:
The line above and below should be `toSatisfyAll`

Reply:
In fact I can fix that ha 😆

Reply:
Oh wait I dont have write access to the branch so I can't all yours

Reply:
Fixed.